composer.json

```
@@ -23,7 +23,7 @@
     "doctrine/inflector": "^1.3",
     "mockery/mockery": "^1.2",
     "phpstan/phpstan": "^0.12",
-    "phpunit/phpunit": "^7.5",
+    "phpunit/phpunit": "^7.5 || ^9.3",
```

```suggestion
    "phpunit/phpunit": "^7.5 || ^9.3",
```

Why not supporting `8.x`, too?